src/workerd/server/alarm-scheduler.c++

```diff
@@ -265,7 +266,8 @@ kj::Promise<void> AlarmScheduler::makeAlarmTask(kj::Duration delay,
       entry.value.backoff++;
       entry.value.retry++;
 
-      entry.value.task = makeAlarmTask(delay, actorRef, scheduledTime);
+      // This alarm task is a retry. As such, we set isRetry to true.
+      entry.value.task = makeAlarmTask(delay, actorRef, scheduledTime, retryNumber++);
```

Comment:
Why not just use `entry.value.countedRetry` for this rather than maintaining a new separate variable?

But if there's some reason to use `retryNumber` instead, don't we want to increment it for the retry *before* passing it in (i.e. `++retryNumber`) rather than after? If I'm reading this right it'd result in both the first attempt of the alarm and the first retry being sent with a retryNumber of 0